README.md

```diff
@@ -26,6 +26,54 @@ Prerequisites:
 Installation (make)
 ------------
 
+### Installing ls1-MarDyn using cmake
+This is the recommended way of building ls1-MarDyn.
+
+#### Configuration
+To build ls1-MarDyn using cmake first create an additional directory on the root ls1-MarDyn directory and change into that directory.
+```bash
+mkdir build
+cd build
+```
+Next, `cmake` has to be executed. In most cases, you will have to specify the compiler with which ls1-MarDyn should be built:
+```bash
+CC=clang CXX=clang++ cmake ..
+```
+Some of the most common compilers are
+| Compiler      | CC    | CXX     |
+|---------------|-------|---------|
+| GCC           | `gcc`   | `g++`     |
+| Intel Classic | `icc`   | `icpc`    |
+| Intel oneAPI  | `icx`   | `icpx`    |
+| Clang         | `clang` | `clang++` |
+
+Specifying the compiler this way is only possible at the first execution of cmake.
+If you want to change the compiler later on, either add another build directory, or first clear the existing build directory.
+
+To configure the options within ls1-MarDyn it is recommended to use `ccmake`:
+```bash
+ccmake .
+```
+That way you can easily edit the available options.
+
+Alternatively, specify the configuration with use of the `cmake` command:
+```bash
+CC=clang CXX=clang++ cmake -DENABLE_MPI=ON ..
+```
+
+Finally, build ls1-MarDyn using:
+```bash
+make
+```
+
+For a faster build, you can make use of parallel building:
+```bash
+make -j $(nproc)
+```
+
+The executable is then found at `build/src/MarDyn`.
+
+
 ### Installing ls1-MarDyn using make
```

Comment:
we should mention that this is only legacy and discouraged.

Reply:
Are all of those platform-specific information still up-to-date?
Is anybody still using the [CRAY](https://github.com/ls1mardyn/ls1-mardyn/blob/master/makefile/cfg/cray-xt-cray.in) or [NEC](https://github.com/ls1mardyn/ls1-mardyn/blob/master/makefile/cfg/nec-sx-ace.in) compilers?

Reply:
> Is the cmake build system polished enough to become the default?

Is there anyone still using the Makefiles?

Yes, I agree we never did a full check if all features are available via CMake but I think everything people currently use is.

Reply:
To do a full check of all features, we could open a new issue and brainstorm and collect all features available via the Makefile. This could also be a kind of (good) documentation of the CMake/Makefile features. I think, there is no overview yet, isn't it?

Reply:
I have created issue #299 regarding the comparison between Makefile and CMake. This PR just switches the position of the CMake and Makefile build instructions, since CMake is "[recommended](https://github.com/ls1mardyn/ls1-mardyn/blob/5bf6608178a8df38fe4db37c0614d972dbd4e97e/README.md?plain=1#L56)". I haven't added this clause.